app/components/delivery-table.js

```diff
@@ -1,9 +1,13 @@
 import Ember from 'ember';
 
 const DeliveryTable = Ember.Component.extend({
-  tagName: 'table',
-  classNames: ['table', 'table-striped', 'table-condensed'],
-  deliveries: null
+  tagName: 'div',
+  columns: [
+    {"propertyName": "project.name" ,"title": "Project Name", "routeName": "deliveries.show"},
```

Comment:
Style question - any reason the keys in these objects are quoted strings?

Reply:
That's just because I copied the style from the docs: http://onechiporenko.github.io/ember-models-table/v.2/#/examples/common-table. Will simplify.